app/models/shipit/task.rb

```diff
@@ -57,7 +57,7 @@ def current
         task.started_at ||= Time.now.utc
       end
 
-      before_transition any => %i(success failed error timedout) do |task|
+      before_transition any => %i(success failed error timedout validating) do |task|
```

Comment:
is there a reason why `aborted` isn't part of this transition?

Reply:
Not all the status you list make sense.

However it would probably be better to take this from the other side:

`after_transition running => any` would be much better and won't break if we introduce more statuses.